### PR TITLE
Added Algolia search

### PR DESCRIPTION
**Pull request by nunix**
Contextual search is set to true

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -55,6 +55,12 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      algolia: {
+        appId: 'Y6AWXACW2K',
+        apiKey: 'c7f1f2eb3086f5d2f2c2631f040263fb',
```
Comment (jimmykarily) [inline docusaurus.config.js:60]: Are these keys only used to query algolia for Epinio text (aka search)? Is it ok to be public like this?

Comment (jimmykarily) [reply docusaurus.config.js:60]: Could someone else used our keys to do search on other indices (see next line) and consume our free quota?

Comment (nunix) [reply docusaurus.config.js:60]: yes, based on Algolia own documentation, these are the "public facing values" and are linked to our account that owns docs.epinio.io.
Therefore, there's no compromise of the account nor a possible way to "add/use" our quota.

**Review by jimmykarily — APPROVED** [review]

**Comment by jimmykarily** [comment]
css is a little broken: 
image

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**